Dockerfile

```diff
@@ -10,5 +10,5 @@ COPY evalytics .
 
 RUN pip install --no-cache-dir -r requirements/${BUILD_ENV}.txt
 
-ENV PYTHONPATH /usr/evalytics
+ENV PYTHONPATH /usr
```

Comment:
@eduardogr this means that from now on that the root of the imports is:

```python
from evalytics.server.models import ...
from evalytics.server.adapters import ...
```

but I can remove that extra path if you prefer